.github/workflows/tests.yml

```diff
@@ -20,3 +20,12 @@ jobs:
         cd ../examples
         ./cpptest.sh
         git diff --exit-code
+  build-meson:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@main
+    - uses: BSFishy/meson-build@master
```

Comment:
I'd prefer to pin to a specific version, looking at them briefly, I think `setup-python@v5` and `BSFishy/meson-builder@v1.0.3`.